libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -16,34 +16,52 @@ export type ButtonType =
     templateUrl: './button.component.html',
     styleUrls: ['./button.component.scss']
 })
-export class ButtonComponent extends BaseComponent {
-    /** Option to make to button compact. */
-    @Input()
-    compact: boolean;
+export class ButtonComponent extends BaseComponent implements AfterViewInit {
 
-    /** The icon to include in the button */
+    /** The icon to include in the button. See the icon page for the list of icons.
+     * Setter is used to control when css class have to be rebuilded.
+     * Default value is set to ''.
+     */
     @Input()
-    glyph: string;
+    public glyph: string;
 
-    /** The type of the button. Types includes
-    'standard','positive', 'negative', 'attention', 'ghost',
+    /** The buttonType of the button. Types includes
+     'standard','positive', 'negative', 'attention', 'ghost',
      'transparent', 'emphasized','menu'.
-     * Leave empty for default (standard button).'*/
+     *Leave empty for default (standard button).'*/
     @Input()
-    type: ButtonType;
+    buttonType: ButtonType;
 
-    /** Tooltip text to show when focused for more*/
+    /** arialabel, tooltip for truncated text
+     * for acccesiblity of the element */
     @Input()
     title?: string;
 
+    /** aria-selected for acccesiblity of the element */
+    @Input()
+    ariaSelected: boolean;
+
+    /** aria-disabled for acccesiblity of the element */
+    @Input()
+    ariaDisabled: boolean;
+
+    /** Specifies a name for the button */
+    @Input()
+    name: string;
+
+    /** Specifies the type of button */
```

Comment:
I think it would be nice to describe that those inputs are setting attributes to the native HTML button

Reply:
Done